### PR TITLE
REST JavaScript client

### PR DESCRIPTION
**Pull request by stefanoborini**
Uses the REST interface to start and stop containers, removing the need for the POST method in HomeHandler.

### CODE REVIEW

**Comment by stefanoborini** [comment]
Fixes #140

File: remoteappmanager/restresources/container.py [hunk 1]
```diff
@@ -4,11 +4,11 @@
 from tornado import gen
 
 from remoteappmanager.docker.docker_labels import SIMPHONY_NS
-from remoteappmanager.handlers.home_handler import _wait_for_http_server_2xx
 from remoteappmanager.rest import exceptions
 from remoteappmanager.rest.resource import Resource
 from remoteappmanager.docker.container import Container as DockerContainer
 from remoteappmanager.utils import url_path_join
+from remoteappmanager.netutils import wait_for_http_server_2xx
```
Comment (kitchoi) [inline remoteappmanager/restresources/container.py]: Could `wait_for_http_server_2xx` be placed in `utils`?

Comment (stefanoborini) [inline remoteappmanager/restresources/container.py]: I prefer it in a separate module that is aimed at network.

**Comment by kitchoi** [comment]
Note that in the HomeHandler on master, if 'wait_for_http_server_2xx' throws a Timeout error, an error message is displayed and the container is removed.  It is no longer the case here, instead the start button "loads" forever.

Original behaviour (try starting 'novnc-never-starts':
screenshot from 2016-07-27 11 05 20

Behaviour for this PR:
screenshot from 2016-07-27 10 54 59

File: tests/test_netutils.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+class TestUtils(AsyncHTTPTestCase):
+    def get_app(self):
+
+        app = web.Application(handlers=[('/short', ShortHandler),
+                                        ('/long', LongHandler)])
+        return app
+
+    @gen_test
+    def test_basic(self):
+        yield wait_for_http_server_2xx(self.get_url("/short"), timeout=2)
```
Comment (kitchoi) [inline tests/test_netutils.py]: how to make sure the request would finish within 2 seconds?

Comment (kitchoi) [inline tests/test_netutils.py]: Never mind that, the computer has to be reaaaaally slow to not have completed the request.

**Comment by kitchoi** [comment]
There is a retrogression on the error handling (see above), but I think we can fix that on a separate PR, all within release 0.6.0?  The rest looks good to merge to me

**Comment by stefanoborini** [comment]
@kitchoi  No I fix it here.

**Comment by stefanoborini** [comment]
Ok, the reason why you are not seeing the error is because we are missing bootstrap. That produces an error in the javascript execution.

In any case, I ported the server side error handling (so it behaves like the old code) but I also need to introduce the javascript error handling to remove the spinner.

**Comment by kitchoi** [comment]
:+1:

**Pull request merged**